Examples/UIExplorer/UIExplorerApp.android.js

```diff
@@ -39,11 +39,52 @@ const UIExplorerStateTitleMap = require('./UIExplorerStateTitleMap');
 
 var DRAWER_WIDTH_LEFT = 56;
 
+function PathActionMap(path: string): ?Object {
```

Comment:
Can we break these out into an independent module, instead of duplicating between `.ios` and `.android`?